utils_nlp/eval/rouge/compute_rouge.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import os
+import shutil
+import time
+import tempfile
+
+from pyrouge import Rouge155
+from rouge import Rouge
+from .rouge_ext import RougeExt
+
+
+def compute_rouge_perl(cand, ref, is_input_files=False, verbose=False):
```

Comment:
for this one, can you add sentence splitting token? I believe this is the common customization that's needed.

Reply:
Can you point me to a script that uses the "sentence splitting token"? I didn't find it in Rouge155 or the BERTSum test_rouge function. I probably missed it.

Reply:
here it splits the sentences by using "<q>": https://github.com/daden-ms/BertSum/blob/master/bertsum/others/pyrouge.py#L254

Reply:
I see. Since this option is not exposed to users by Rouge155, we won't be able to control it without changing the underlying pyrouge package. I prefer not changing the underlying pyrouge package if possible. It's better to take dependency on a standard package instead of creating our own variation for a small change like this.
I think BERTSum did it specifically for CNN/DailyMail. Is it possible to preprocess CNN/DM to use <\n> as line separator instead of <q>?

Reply:
The harvardnlp CNN/DM uses <q> in the summary but we can replace this with a new line when calculating rouge score. The result is close enought, as indicated by https://github.com/google-research/google-research/issues/168